src/helper/sendEmail.js

```diff
@@ -1,32 +1,34 @@
 import nodemailer from 'nodemailer';
 import 'dotenv/config';
 import {google} from 'googleapis';
+import sgMail from '@sendgrid/mail'
 import emailTemplate from './emailTemplate';
-const oAuth2Client=new google.auth.OAuth2(process.env.CLIENT_ID,process.env.CLIENT_SECRET,process.env.REDIRECT_URL);
-oAuth2Client.setCredentials({refresh_token:process.env.REFRESH_TOKEN});
+
+sgMail.setApiKey(process.env.SENDGRID_API_KEY)
 
 const sendEmail = async (userInfo) =>{
-  const accessToken= await oAuth2Client.getAccessToken()
-  const transporter = nodemailer.createTransport({
-      service: 'gmail',
-      auth: {
-        type:'OAuth2',
-        user: process.env.GMAIL_EMAIL,
-        clientId:process.env.CLIENT_ID,
-        clientSecret:process.env.CLIENT_SECRET,
-        refreshToken:process.env.REFRESH_TOKEN,
-        accessToken:accessToken
+  // const accessToken= await oAuth2Client.getAccessToken() 
+  // const transporter = nodemailer.createTransport({
+  //     service: 'gmail',
+  //     auth: {
+
+  //       type:'OAuth2',
+  //       user: process.env.GMAIL_EMAIL,
+  //       clientId:process.env.CLIENT_ID,
+  //       clientSecret:process.env.CLIENT_SECRET,
+  //       refreshToken:process.env.REFRESH_TOKEN,
+  //       accessToken:accessToken,
 
-      }
-    });
+  //     }
+  //   });
```

Comment:
Please remove this, and any other sections of commented-out code

Reply:
going to do it too